.golangci.yml

```diff
@@ -0,0 +1,20 @@
+linters:
+  enable:
+    - structcheck
+    - varcheck
+    - staticcheck
+    - unconvert
+    - gofmt
+    - goimports
+    - revive
+    - ineffassign
+    - vet
+    - unused
+    - misspell
+    - gofumpt
+  disable:
+    - errcheck
```

Comment:
yeah, we should eventually enable this too :)

Reply:
yeah, if we need more more linters, just add them in .golangci.yml.